frontend/src/components/TextArea/index.tsx

```diff
@@ -45,10 +48,14 @@ export const TextArea: React.FC<TextInputProps> = ({
                 className={classNames(
                     'absolute select-none left-0 top-0 pl-4 peer-placeholder-shown:top-3.5',
                     'text-primary text-xs peer-placeholder-shown:text-neutral-10 peer-placeholder-shown:text-base',
-                    'transition-all'
+                    'transition-all',
+                    {
+                        'text-red peer-placeholder-shown:text-red':
+                            touched && errors,
+                    }
                 )}
             >
-                {touched && value ? label : placeholder}
+                {touched && errors ? errors : value ? label : placeholder}
```

Comment:
Don't nest multiple ternary expressions without parens, it's hard to see what is the actual result -- especially as the order varies between languages (or well, at least one special language)

https://i.imgur.com/1zgFd.jpg